packages/sirius-web/backend/sirius-web-services/src/main/java/org/eclipse/sirius/web/services/representations/RepresentationsDescriptionProvider.java

```diff
@@ -59,13 +67,17 @@ public class RepresentationsDescriptionProvider implements IRepresentationsDescr
 
     private final IRepresentationService representationService;
 
+    private final IRepresentationSearchService representationSearchService;
```

Comment:
Do you really need to load the representations, couldn't using their metadata be enough? (I know, using the metadata also loads the representations right now but at least it does not parse them)

Reply:
I only use it to load "sub-portals", and for them I do need the actual representations to expose their internal structure in the tree (see `getPortalChildren`).